github3/models.py

```diff
@@ -202,11 +202,6 @@ def _patch(self, url, **kwargs):
     def _post(self, url, data=None, json=True, **kwargs):
         if json:
             data = dumps(data) if data is not None else data
-        elif 'headers' in kwargs:
-            # Override the Content-Type header
-            kwargs['headers'] = {
```

Comment:
Oh this is such a subtle bug. I think I probably wrote this while I was working predominantly in Ruby-land.

I think this should be

```py
elif 'headers' in kwargs:
   new_headers = {'Content-Type': None}
   new_headers.update(kwargs['headers'])
   kwargs['headers'] = new_headers
```

I think the idea was to force the `Content-Type` to be unset if we overrode headers, but I'm not sure we want that anyway.